src/locales/en.json

```diff
@@ -886,6 +886,30 @@
     "title": "Cost Management overview",
     "title_aria_label": "A description of OpenShift on AWS, OpenShift, AWS, and Azure"
   },
+  "page_cost_models": {
+    "assigned_sources": "Assigned sources",
+    "aws": "Amazon Web Services",
+    "azure": "Microsoft Azure",
+    "can_delete_description": "This action will delete {{ name }} cost model from the system. This action cannot be undone",
+    "cancel": "Cancel",
+    "cannot_delete_description_body": "The following sources are assigned to {{ name }} cost model:",
+    "cannot_delete_description_head": "You must unassign any sources before you can delete this cost model.",
+    "create_cost_model": "Create cost model",
+    "delete": "Delete",
+    "delete_cost_model": "Delete cost model",
+    "delete_title": "Delete cost model",
+    "description": "Description",
+    "filter_by_name": "Filter by name",
+    "filter_by_source_type": "Filter by source type",
+    "header_popover": "A cost model allows you to associate a price to metrics provided by your sources to charge for utilization of resources.",
+    "header_title": "Cost models",
+    "last_change": "Last change",
+    "name": "Name",
+    "no_cost_models_description": "Create a cost model to start calculating your hybrid cloud costs using custom price lists, markups, or both. Click on the button below to begin the journey.",
```

Comment:
It looks like a lot of these strings have been duplicated in en.json. Are both still in use?

We've been trying to clean up dead strings, so want to ensure we're not leaving unused strings elsewhere.

Reply:
Right, I will remove old keys :+1: